shared/actions/tracker.js

```diff
@@ -184,6 +185,10 @@ function registerIdentifyUi(): TrackerActionCreator {
         let trackerTimeoutError = 0
 
         const onStart = username => {
+          // Don't do this on mobile
```

Comment:
Android complains about long timeouts (over 1m) and likely this doesn't make sense in the mobile context anyways. its not a tracker card that hangs out on the side, it'd be your whole screen. no one is waiting 5 minutes...